test/bin/scenario.sh

```diff
@@ -224,7 +224,8 @@ sos_report_for_vm() {
 }
 
 invoke_qemu_script() {
-    "${ROOTDIR}/_output/robotenv/bin/python" "${ROOTDIR}/test/resources/qemu-guest-agent.py" "$@"
+    timeout --verbose --foreground 2m \
```

Comment:
This does not seem to be related to the current PR.
Please, rebase the code.

Reply:
sorry, my bad !! Let me do that.